### PR TITLE
Test cases for `ArgminMinMax`

### PR DESCRIPTION
**Pull request by Shreyan11**
Please do let me know if there's anything else to be done,I'm open to all suggestions.

### CODE REVIEW

File: README.md
Comment (stefan-k) [inline README.md:1]: There are still changes to this file which should not be there.

File: argmin-math/src/primitives/minmax.rs [hunk 1]
```diff
@@ -49,3 +49,27 @@ make_minmax!(Complex<isize>);
 make_minmax!(Complex<usize>);
 
 // TODO: tests!!!
```
Comment (stefan-k) [inline argmin-math/src/primitives/minmax.rs:51]: Please remove this line.

**Review by stefan-k — CHANGES_REQUESTED** [review]
Thank you! Please don't open a new PR for each review, you can do all changes in one PR. There are still unnecessary changes to `README.md`, probably due to a failed rebase.

File: argmin-math/src/primitives/minmax.rs (hunk 1 above)
Comment (Shreyan11) [reply argmin-math/src/primitives/minmax.rs]: I've updated it,it should be good,let me know if there's anything else to be done

**Review by stefan-k — APPROVED** [review]
Looks good to me! Thanks!

**Pull request merged**